### DISCUSSION TITLE
How to access the CMS Page

### DISCUSSION THREAD

**Discussion by saxenachaitanya** [CaiJimmy/hugo-theme-stack#348] [Q&A]
Hello

I want to write the posts but I cannot able to access the CMS of the website how can I access the CMS of the wesbite

**Comment by MikDal002** [comment]
This repo is Hugo's template, so the CMS "is done" by [Hugo](https://gohugo.io/), but there is not an ordinary "CMS" it's rather a set of files which represent your articles. Then Hugo creates a static page with selected layout and this repository is about layout only.

**Reply by saxenachaitanya** [reply]
Ok soo how can I access the admin panel of the website?

**Reply by MikDal002** [reply]
There is no admin panel, precisely how it works you can find in Hugo documentation (this page should give you all necessary information to get started https://gohugo.io/getting-started/quick-start/). Workflow with Hugo looks like that:
1. Write your text in a file (e.g. markdown, asciidoctor or even HTML), 
2. put in appropriate Hugo's folder (usually `content`),
3. start Hugo (e.g. by shell command `hugo serve` - you need to install Hugo first of course) 
4. See your results.
When you are done, then you can publish your website somewhere (e.g. netlify). 

I'm not going into detailed instruction because everything what is needed to start is in documentation (links above), so won't copy that here. If you meet any problems with getting started, feel free to write down here. 

Please, note that Hugo is addressed rather to more technical person with specific requirements rather than those who want just focus on writing – if you are that second type of person than WordPress might be more appropriate – it took me a few days to start working with Hugo as I wanted to.
